great_expectations/core/usage_statistics/schemas.py

```diff
@@ -750,6 +750,135 @@
     "additionalProperties": False,
 }
 
+anonymized_domain_builder_schema = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "anonymized-domain-builder",
+    "definitions": {
+        "anonymized_string": anonymized_string_schema,
+        "anonymized_batch_request": anonymized_batch_request_schema,
+        "anonymized_datasource_name": anonymized_datasource_name_schema,
```

Comment:
No such luck narrowing this down @anthonyburdi @alexsherstinsky - if the definition is used anywhere in the hierarchy of the given object, it must be included in the definitions keys.

I'll keep reading the JSON Schema guide to see if we have a better solution but this seems to track with the rest of the schemas noted herein.